app/controllers/submissions_controller.rb

```diff
@@ -118,6 +132,36 @@ def destroy
                                                    @submission.assessment)) && return
   end
 
+  action_auth_level :destroyBatch, :instructor
+  def destroy_batch
```

Comment:
there needs to be authentication here to ensure that an instructor destroying the submissions here is an instructor of this course. there also needs to be validation that the submissions here belong to the assessment as well.